weapons/allied-infantry.yaml

```diff
@@ -173,3 +173,22 @@ AWPE:
 20mme:
 	Inherits: 20mm
 	Burst: 2
+
+NeutronRifle:
+	ValidTargets: Ground
+	ReloadDelay: 120
+	Range: 10c0
+	MinRange: 0c001
```

Comment:
That shouldn't be needed here - jumpjet stuff can crash at range 0 for some reason. CLeg ain't jumpjet.